index.d.ts

```diff
@@ -18,7 +22,7 @@ declare const humanizeString: {
 	//=> 'Foo bar'
 	```
 	*/
-	(text: string): string;
+	(text: string, opts: HumanizeProps): string;
```

Comment:
Follow this style guide for the TS changes: https://github.com/sindresorhus/typescript-definition-style-guide